packages/client/hmi-client/src/services/model-configurations.ts

```diff
@@ -235,3 +237,47 @@ export function getMissingInputAmount(modelConfiguration: ModelConfiguration) {
 	});
 	return missingInputs;
 }
+
+export function getMmtParameters(modelConfiguration, mmt, mmtParams) {
+	const collapsedParameters = collapseParameters(mmt, mmtParams);
+	const parameters = getParameters(modelConfiguration);
+	return Array.from(collapsedParameters.keys()).map((id) => {
```

Comment:
Can you elaborate on what we are trying to do here? `collapseParameters` and likewise, `collapseInitials` are used for building matrices - they are heuristically based so do not always give you the right answer, and I think probably not what you are looking for in the context of extracting model-config from a document.

Moreover, if you want to extract information from MMT and the functions do not exist, they would be better placed somewhere in `src/model-representation/mira`

Reply:
removed collapseParameter() since it was not the right method to use in this situation.  Now using `service.ts`getParameters() & getInitials()`